app/models/comment.rb

```diff
@@ -1,7 +1,7 @@
 class Comment < ApplicationRecord
   belongs_to :subject
 
-  after_create :push_to_channels
+  after_save :push_to_channels
```

Comment:
This could potentially create a LOT more background jobs, some of which may be no-ops, we should make sure that we only push to channels if the visible contents of the comment has changed, see https://github.com/octobox/octobox/blob/master/app/models/notification.rb#L190-L196